samza-core/src/main/java/org/apache/samza/clustermanager/StandbyContainerManager.java

```diff
@@ -126,6 +126,30 @@ public void handleContainerLaunchFail(String containerID, String resourceID,
     }
   }
 
+  /**
+   *  Handle the failed stop for a container, based on
+   *  Case 1. If it is standby container, continue the failover
+   *  Case 2. If it is an active container, then log a warning and return.
+   * @param containerID the ID of the container that has failed
```

Comment:
nit: @param containerID logical id ex 0,1,2 of the failed samza container
       @param resourceID physical id of the failed container